tests/test_venue_submission.py

```diff
@@ -439,4 +439,17 @@ def test_decision_stage(self, venue, openreview_client, helpers):
 
     def test_post_decision_stage(self, venue, openreview_client):
 
-        venue.post_decision_stage()
\ No newline at end of file
+        venue.post_decision_stage()
+
+    def test_submission_revision_stage(self, venue, openreview_client):
+        now = datetime.datetime.utcnow()
+        venue.submission_revision_stage = openreview.SubmissionRevisionStage(
```

Comment:
can you define this at the beginning of the tests with the other stages and then test the cdate process function runs correctly?